packages/renderer/src/lib/preferences/PreferencesRenderingItem.svelte

```diff
@@ -56,30 +61,51 @@ function updateResetButtonVisibility(recordValue: any) {
 function doResetToDefault() {
   resetToDefault = true;
 }
+
+function openGitHubDiscussion(): void {
+  if (!recordUI.experimental?.githubDiscussionLink) return;
+
+  window.openExternal(recordUI.experimental.githubDiscussionLink).catch(console.error);
+}
 </script>
 
-<div class="flex flex-row px-2 py-2 justify-between w-full text-[color:var(--pd-invert-content-card-text)]">
-  <div
-    class="flex flex-col {recordUI.original.type === 'string' &&
+<div class="flex flex-col px-2 py-2 w-full text-[color:var(--pd-invert-content-card-text)] space-y-2">
```

Comment:
The html diff look crazy because I changed the indentation to encapsulate everything in a div... so it look like I changed everything, but I didn't :(